### ISSUE TITLE
Incorporate jekyll_pages_api_search

### ISSUE THREAD

**Issue by line47** [18F/uswds-jekyll#34]
Incorporate [jekyll_pages_api_search](https://github.com/18F/jekyll_pages_api_search) into the theme to allow for simple searches to be performed on the site.

**Comment by line47** [comment]
I'm getting this error when trying to run on https://github.com/18F/frontend

```
TypeError: Cannot set property 'value' of null
    at SearchUi.renderResults (:4000/assets/js/search-bundle.js:5)
    at :4000/assets/js/search-bundle.js:8
    at <anonymous>
(anonymous) @ :4000/assets/js/search-bundle.js:8
```

**Comment by line47** [comment]
I was missing the proper id for search which should be `id="search-input"`, it's working now! 🎉

**Comment by afeld** [comment]
Closed by #32.
